docs/source/install.md

```diff
@@ -0,0 +1,137 @@
+# Install Fabric Samples
+
+Please install the [Prerequisites](./prereqs.html) before following these install instructions.
+
+**Fabric-Samples:**<br>
+We think the best way to understand something is to use it yourself.  To help you use Fabric, we have created a simple Fabric Network called `test-net` and a set of sample applications that demonstrate its core capabilities. We have also precompiled `Fabric CLI tool binaries` and `Fabric Docker Images` which will be downloaded to your environment, to get you going.
+
+**Optional Fabric Download:**<br>
+We are huge proponents of reviewing actual code whereever possible to get a solid understanding of technical concepts. If you feel the same way, we have added instructions to help you download the Fabric code base on to your system.  This is a personal preference, you do not need this code to run any of the samples.
+<br>
+<br>
+
+#### Mac
+
+> **Fabric-Samples:**<br>
+> 
+>> **Install**  Install Fabric-Samples in `$HOME/go/src/github.com/<your_github_userid>` directory.  This is a Golang Community recommendation for Go projects and this location will also enable local directories to be shared with Docker containers
+>> ```shell
+>> $ mkdir -p $HOME/go/src/github.com/<your_github_userid>
```

Comment:
Is it necessary to clone the repo's separately, since the bootstrap.sh command already does that? or are we changing the flow? (cloning the repo does make things more deliberate, I can image people running the command from anywhere accidentally)

is the `go/src/` path necessary? That sound required for developers? also, the samples contains more scripts and javascript than any go code.

Reply:
Answered the comment down below for brevity.